src/Particular.CodeRules/AnalyzerReleases.Shipped.md

```diff
@@ -0,0 +1,23 @@
+﻿## Release 0.1
```

Comment:
To address this new violation:

> error RS2008: Enable analyzer release tracking for the analyzer project containing rule 'PCR0002'

Reply:
Are these files actually used in some way by the analyzer, or otherwise surfaced in some way? Seems like an odd thing to enforce if they are just internal repo artifacts.

Reply:
I have no idea. I just did whatever was required to satisfy the analyser.